### PR TITLE
Revamp .NET Framework landing page

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #17502

[Internal preview link.](https://review.docs.microsoft.com/en-us/dotnet/framework/?branch=pr-en-us-17540)

### CODE REVIEW

File: docs/framework/index.yml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+  # Card
+  - title: Create Windows Forms apps
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Overview of Windows Forms
+            url: winforms/index.md
+      - linkListType: how-to-guide
+        links:
```
Comment (tdykstra) [inline docs/framework/index.yml]: These seem too narrowly focused for landing page links.  Is there no getting starting tutorial that we can link to, like the Create WPF Apps card has?

File: docs/framework/index.yml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+          - text: Add and remove items from ComboBox, ListBox, and CheckedListBox controls 
+            url: winforms/controls/add-and-remove-items-from-a-wf-combobox.md
+          - text: Make thread-safe calls to controls
+            url: winforms/controls/how-to-make-thread-safe-calls-to-windows-forms-controls.md
+      - linkListType: reference
+        links:
+          - text: Design-time errors in the Windows Forms Designer
```
Comment (tdykstra) [inline docs/framework/index.yml]: This doesn't seem to be the kind of doc I'd look for in a landing page.

File: docs/framework/index.yml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            url: winforms/controls/design-time-errors-in-the-windows-forms-designer.md
+
+  # Card
+  - title: Work with data using ADO.NET
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Overview of ADO.NET
+            url: data/adonet/ado-net-overview.md
+      - linkListType: concept
```
Comment (tdykstra) [inline docs/framework/index.yml]: The links in this section seem more like how-to docs than conceptual docs, except perhaps the connection pooling doc

File: docs/framework/index.yml [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+            url: winforms/controls/add-and-remove-items-from-a-wf-combobox.md
+          - text: Make thread-safe calls to controls
+            url: winforms/controls/how-to-make-thread-safe-calls-to-windows-forms-controls.md
+      - linkListType: reference
+        links:
+          - text: Design-time errors in the Windows Forms Designer 
+            url: winforms/controls/design-time-errors-in-the-windows-forms-designer.md
+
+  # Card
+  - title: Work with data using ADO.NET
```
Comment (tdykstra) [inline docs/framework/index.yml]: Is there no getting started tutorial we could put in a tutorials section?

**Review by tdykstra — APPROVED** [review]
Looks good. I added some thoughts for you to consider regarding what seemed to me to fit or not fit in the various categories.

File: docs/framework/index.yml [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+# Cards and links should be based on top customer tasks or top subjects
+# Start card title with a verb
+  # Card (optional)
+  - title: Learn about .NET Framework
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: .NET Framework overview
+            url: get-started/overview.md
+          - text: Migrate from .NET Framework to .NET Core
```
[the remark this answers is not in the document]
Comment (gewarren) [reply docs/framework/index.yml]: "Overview" refers to the type of content, and that page is an overview. I thought we might add that link in here as a clue to the fact that .NET Framework is no longer the latest and greatest. Do you have a suggestion for this link, e.g. remove it, change the linkListType, move it to a different card?

File: docs/framework/index.yml [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+            url: ../core/porting/index.md
+      - linkListType: reference
+        links:
+          - text: System requirements
```
[the remark this answers is not in the document]
Comment (gewarren) [reply docs/framework/index.yml]: Good catch.

File: docs/framework/index.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (tdykstra) [reply docs/framework/index.yml]: I would remove it or make a separate "Migrate to .NET Core" card to hold it.

**Pull request merged**